src/Configuration/src/SpringBootBase/SpringBootConfigurationBuilderExtensions.cs

```diff
@@ -15,19 +17,30 @@ public static class SpringBootConfigurationBuilderExtensions
         /// Configuration keys in '.' delimited style are also converted to a format understood by .NET
         /// </summary>
         /// <param name="builder">the configuration builder</param>
-        /// <param name="loggerFactory">the logger factory to use</param>
         /// <returns>builder</returns>
-        public static IConfigurationBuilder AddSpringBootEnv(this IConfigurationBuilder builder, ILoggerFactory loggerFactory = null)
```

Comment:
This was not getting added anywhere and is particularly difficult during ConfigureAppConfiguration step. The typical way microsoft themselves report errors is by throwing for malformed configuration.